src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibLeaseCoordinator.java

```diff
@@ -292,6 +292,20 @@ public Checkpoint getCheckpointObject(String shardId) throws KinesisClientLibExc
         }
     }
 
+    public Set<String> getChildShardIds(String shardId) {
+        String errorMessage = "Unable to fetch childShardIds for shardId " + shardId;
+        try {
+            KinesisClientLease lease = leaseManager.getLease(shardId);
+            if (lease == null) {
+                throw new KinesisClientLibIOException(errorMessage);
```

Comment:
Maybe we should just log here, looking at other usages of KinesisClientLibIOException we use this for inconsistent shard info edge cases, I don't think that us losing a lease is unusual behavior here, we can just log that this worker doesn't own this lease.